team_ssh_keys.sh

```diff
@@ -23,11 +22,8 @@ if [ -z "$OWNER_ALIASES" ]; then
 fi
 
 # shellcheck disable=SC2016
-MEMBERS=$(echo "$OWNER_ALIASES" | yq --arg team_name "$TEAM_NAME" -r '.aliases[$team_name] | join(" ")')
+MEMBERS=$(yq --arg team_name "$TEAM_NAME" -r '.aliases[$team_name] | join(" ")' <<< "$OWNER_ALIASES")
 
-TMP_FILE=$(mktemp)
 for member in $MEMBERS; do
-    curl -s "https://github.com/$member.keys" >> "$TMP_FILE"
+    curl -s "https://github.com/$member.keys"
```

Comment:
That's great. How about we also add the github username as a comment? Something like:
```
key=$(curl -s "https://github.com/$member.keys")                                                                                                                      
echo "$key $member"
```

Reply:
That's a valuable addition for the next PR imho :+1: